docs/workers.md

```diff
@@ -218,6 +219,7 @@ information.
     ^/_matrix/client/(api/v1|r0|v3|unstable)/voip/turnServer$
     ^/_matrix/client/(api/v1|r0|v3|unstable)/rooms/.*/event/
     ^/_matrix/client/(api/v1|r0|v3|unstable)/joined_rooms$
+    ^/_matrix/client/v1/rooms/.*/timestamp_to_event$
```

Comment:
I don't fully understand why we have both `api/v1/` and `v1/`.

But since `/threads` is already using `/v1` I'm happy with it.

Reply:
Some legacy endpoints used `api/v1` instead of `r0`, this is why those endpoints all moved to `v3` when the new endpoint versioning scheme came out (instead of moving to `v1`, which can easily be confused with `api/v1`). Any *new* endpoints do get added as `v1` one though.

Reply:
Can we remove `api/v1` and `v3` at some point? Is there something to track to reconcile this? Are there Synapse docs for this (would make some great docs)?

 - `r0` was the thing before: https://matrix.org/docs/spec/client_server/r0.6.1.html
 - `v1` is the thing now: https://spec.matrix.org/v1.4/client-server-api